sdk/src/SDK.ts

```diff
@@ -166,7 +165,8 @@ export class SDK {
     /* istanbul ignore next */
     public error = (response: Response): Promise<string> => this.client().error(response);
 }
-
+// eslint-disable-next-line no-console
+console.warn('Sandbox support is deprecated. Please migrate your application to Production Server.');
```

Comment:
Let's add the warn in constructor. And only console when developer pass sandbox uri